src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1074,9 +1075,8 @@ sub check_star_catalog {
     my $min_z = $z_ang_max;
 
     my $dither;			# Global dither for observation
-    if (defined $self->{DITHER_Y_AMP} and defined $self->{DITHER_Z_AMP}) {
-	$dither = ($self->{DITHER_Y_AMP} > $self->{DITHER_Z_AMP} ?
-		   $self->{DITHER_Y_AMP} : $self->{DITHER_Z_AMP}) * 3600.0;
+    if (defined $self->{cmd_dither_y_amp} and defined $self->{cmd_dither_z_amp}) {
+	$dither = max($self->{cmd_dither_y_amy}, $self->{cmd_dither_z_amp});
```

Comment:
No plan in mind.  If I were, say, developing a stateful checker, I'd stay aware of checking the two star catalogs with separate states and parameters.